gnat2goto/driver/tree_walk.adb

```diff
@@ -113,13 +115,20 @@ package body Tree_Walk is
    with Pre  => Nkind (N) = N_Function_Call,
         Post => Kind (Do_Function_Call'Result) in Class_Expr;
 
+   function Do_Assume (N : Node_Id; Assumption : Irep) return Irep;
+
+   function Do_Assert (N : Node_Id; Assertion : Irep; Description : Irep)
+                       return Irep;
+
+   function Do_Range_Assert (N : Node_Id; Value : Irep; Value_Type : Irep)
```

Comment:
These 3 functions actually generate code, but they're using the naming scheme of the functions that handle an AST node. I think we need to come up with less confusing names for these.